common-pg/src/main/scala/com/socrata/pg/soql/SoQLAnalysisSqlizer.scala

```diff
@@ -461,7 +474,18 @@ trait SoQLAnalysisSqlizer {
       ana.limit.map(" LIMIT " + _.toString).getOrElse("") +
       ana.offset.map(" OFFSET " + _.toString).getOrElse("")
 
-    val result = ParametricSql(Seq(countBySubQuery(analysis)(reqRowCountFinal, completeSql)), setParamsOrderBy)
+    val result = ParametricSql(Seq(countBySubQuery(analysis)(reqRowCountFinal, completeSql)),
+                               setParams ++
+                                 setParamsSelect ++
+                                 finalSetParamsJoin ++
+                                 setParamsWhere ++
+                                 setParamsWhereSearch ++
+                                 setParamsGroupBy ++
+                                 setParamsHaving ++
+                                 setParamsHavingSearch ++
+                                 setParamsOrderBy ++
+                                 setParamsOrderBy)
```

Comment:
setParamsOrderBy added twice

Reply:
Yeah, that's one of the test failures (two of the test failures, actually).  The others are in soql-reference so I'll be making a new PR there too.